akka-projection-grpc/src/main/scala/akka/projection/grpc/replication/javadsl/ReplicatedBehaviors.scala

```diff
@@ -17,7 +16,6 @@ import akka.persistence.typed.javadsl.ReplicationContext
  *
  * Can optionally be composed with other Behavior factories, to get access to actor context or timers.
  */
-@ApiMayChange
 abstract class ReplicatedBehaviors[Command, Event, State] {
```

Comment:
```
* Not for user extension
*/
@DoNotInherit
```

Both language versions, we implement it and hand it to the user who only uses it